webapp/src/app/pages/components/knowledge-graph/concept-map/concept-map.component.ts

```
@@ -267,33 +267,27 @@ export class ConceptMapComponent {
     private changeDetectorRef: ChangeDetectorRef, // avoids errors when property changed after being checked
     private store: Store<State>,
-    private socket:Socket
+    private socket: Socket
   ) {
     // get current user
     this.subscriptions.push(
       this.store
-      .select(getLoggedInUser)
-      .subscribe((user) => (this.loggedInUser = user))
+        .select(getLoggedInUser)
+        .subscribe((user) => (this.loggedInUser = user))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)